src/appkit.cs

```diff
@@ -6187,6 +6188,7 @@ interface NSFontChanging
 	[BaseType (typeof (NSObject))]
 	[DisableDefaultCtor] // crash at runtime (e.g. description). Documentation state: "You don’t create NSFont objects using the alloc and init methods."
 	partial interface NSFont : NSSecureCoding, NSCopying {
+		[ThreadSafe]
```

Comment:
same

> Font objects are immutable, so it is safe to use them from multiple threads in your app.
https://developer.apple.com/library/archive/documentation/TextFonts/Conceptual/CocoaTextArchitecture/FontHandling/FontHandling.html

and `NSFont` is mention in the same paragraph.